ui/app/helpers/utils/metametrics.util.js

```diff
@@ -138,7 +138,7 @@ function composeUrl (config) {
     numberOfAccounts,
     version,
     previousPath = '',
-    currentPath,
+    currentPath = '',
```

Comment:
This may (always?) be `undefined` when we send a metrics event from the background with the UI closed.

Reply:
Yep, and as a result, the background metametrics have never actually worked. Not a single event recorded in Matomo from the background. Awesome.

Reply:
Oof 😞

Reply:
Should we keep this?

Reply:
I don't think so - it should be completely optional now, so a default isn't required

Reply:
Fixed in: 70b50a0